tests/core/pyspec/eth2spec/utils/ssz/ssz_impl.py

```diff
@@ -8,6 +8,8 @@
 def serialize(obj: View) -> bytes:
     return obj.encode_bytes()
 
+def deserialize(cls: View, data: bytes) -> object:
```

Comment:
Not sure if `View` is the right type for a View class
```suggestion
def deserialize(cls, data: bytes) -> object:
```

Reply:
Also, maybe the return type should be `View`.

Reply:
I looked at remerkleable & ascertained that this should take `Type[View], bytes` and return `View`.

https://github.com/protolambda/remerkleable/blob/1099d0ab038ea25ece506bbed1e8357aba1295be/remerkleable/test_impl.py#L425-L439

I pushed this commit to fix with this change: https://github.com/ethereum/consensus-specs/pull/3976/commits/0ecc87164b0b09f9659800ab45214408d00f8028

Reply:
https://brainstorm.pinet.com/project/66a8bfda349c75005278ea87